eng/Version.Details.xml

```diff
@@ -171,9 +171,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>474307e526160c813c9fd58060eb8356ccca6099</Sha>
     </Dependency>
-    <Dependency Name="Private.SourceBuild.ReferencePackages" Version="1.0.0-beta.21457.1">
+    <Dependency Name="Private.SourceBuild.ReferencePackages" Version="1.0.0-beta.21460.1">
       <Uri>https://github.com/dotnet/source-build-reference-packages</Uri>
-      <Sha>203ced7252f5104d13b83bf446a6c6dea827b673</Sha>
+      <Sha>896532ec53ea317e3136ad3849ef1944a31b9f6b</Sha>
```

Comment:
Can you add `<SourceBuild RepoName="source-build-reference-packages" ManagedOnly="true" />` to this?  We will need it when we migrate over to using this file vs the SB specific one.

Reply:
Thanks.  I added it.